recipe/meta.yaml

```diff
@@ -56,19 +56,22 @@ build:
   string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
   run_exports:
     - {{ pin_subpackage('hdf5', max_pin='x.x.x') }} {{ build_pin }}
+  merge_build_host: true  # [win]
 
 requirements:
   build:
     - gnuconfig  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}  # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
```

Comment:
I don't think we need this. what does netcdf4-fortran do?

Reply:
netcdf-fortran is built with C and Fortran MinGW compilers and VS to link shared lib

Reply:
I try without at beginning but it failed